documentation/docs/03-runtime/07-svelte-action.md

```diff
@@ -2,7 +2,79 @@
 title: svelte/action
 ---
 
-TODO
+Actions are functions that are called when an element is created. They can return an object with a `destroy` method that is called after the element is unmounted:
+
+```svelte
+<!--- file: App.svelte --->
+<script>
+	/** @type {import('svelte/action').Action}  */
+	function foo(node) {
+		// the node has been mounted in the DOM
+
+		return {
+			destroy() {
+				// the node has been removed from the DOM
+			}
+		};
+	}
+</script>
+
+<div use:foo />
+```
+
+An action can have a parameter. If the returned value has an `update` method, it will be called whenever that parameter changes, immediately after Svelte has applied updates to the markup.
```

Comment:
```suggestion
An action can have a parameter. If the returned value has an `update` method, it will be called immediately after Svelte has applied updates to the markup whenever that parameter changes.
```